src/main/g8/pom.xml

```diff
@@ -0,0 +1,52 @@
+<!-- #build-sample -->
+<project>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>hello-akka-java</groupId>
+    <artifactId>app</artifactId>
+    <version>1.0</version>
+
```

Comment:
I think you can add `<properties>` and define `akka.version` and use that with `${akka.version}` in dependencies